src/components/RepoBadge/RepoBadge.js

```diff
@@ -0,0 +1,73 @@
+import React, { useMemo } from 'react'
+import { blockExplorerUrl, Link } from '@aragon/ui'
+import { network } from '../../environment'
+import AppIcon from '../../components/AppIcon/AppIcon'
+import { KNOWN_ICONS, isKnownRepo } from '../../repo-utils'
+import { repoBaseUrl, parseHub } from '../../url-utils'
+
+function useRepoBadge(repo) {
+  const {
+    content: { appId, contractAddresses, name, icons },
+    version,
+  } = repo
+
+  return {
+    baseUrl: repoBaseUrl(repo),
+    icons: icons,
+    contractAddress: contractAddresses,
+    knownIcon: KNOWN_ICONS.has(appId) ? KNOWN_ICONS.get(appId) : null,
+    name: name,
+  }
+}
+
+const RepoBadge = ({ repo }) => {
+  const { baseUrl, icons, contractAddress, knownIcon, name } = useRepoBadge(
+    repo
+  )
+
+  return (
+    <div
+      css={`
+        display: inline-flex;
+        align-items: center;
+        min-width: 0;
+      `}
+    >
+      <Link
+        target="_blank"
+        href={blockExplorerUrl('address', contractAddress, {
+          networkType: network.type,
+        })}
+        css={`
+          display: flex;
+          background: #daeaef;
+          border-radius: 3px;
+          align-items: center;
+          height: 22px;
```

Comment:
```suggestion
          height: ${3 * GU}px;
```

Currently there's a small height difference between the icon and text:

<img width="91" alt="Screen Shot 2019-08-26 at 8 21 35 PM" src="https://user-images.githubusercontent.com/4166642/63713013-202fa900-c83f-11e9-8aaa-ee17445d3fe0.png">

-------

We should also update the text colour so that it's normal content, and apply overflow: hidden here (and change the `radius` declaration in `AppIcon` to be 0) to follow the design:

<img width="114" alt="Screen Shot 2019-08-26 at 8 22 40 PM" src="https://user-images.githubusercontent.com/4166642/63713103-448b8580-c83f-11e9-9f1a-ba8a22bf9ece.png">